libs/application/templates/rental-agreement/src/forms/rentalPeriod/rentalPeriodSecurityDeposit.ts

```diff
@@ -2,20 +2,222 @@ import {
   buildSubSection,
   buildMultiField,
   buildTextField,
+  buildSelectField,
+  buildDescriptionField,
+  buildAlertMessageField,
+  buildHiddenInputWithWatchedValue,
 } from '@island.is/application/core'
 
+import * as m from '../../lib/messages'
+import { FormValue } from '@island.is/application/types'
+
+const messages = m.prerequisites.intro
+
+const securityDepositTypes = [
+  {
+    label: m.securityDeposit.typeSelectionBankGuaranteeTitle,
+    value: 'bankGuarantee',
+  },
+  {
+    label: m.securityDeposit.typeSelectionCapitalTitle,
+    value: 'capital',
+  },
+  {
+    label: m.securityDeposit.typeSelectionThirdPartyGuaranteeTitle,
+    value: 'thirdPartyGuarantee',
+  },
+  {
+    label: m.securityDeposit.typeSelectionInsuranceCompanyTitle,
+    value: 'insuranceCompany',
+  },
+  {
+    label: m.securityDeposit.typeSelectionMutualFundTitle,
+    value: 'mutualFund',
+  },
+  {
+    label: m.securityDeposit.typeSelectionOtherTitle,
+    value: 'other',
+  },
+]
+
+const rentalAmountDetails = [
+  {
+    label: m.securityDeposit.amountSelection1Month,
+    value: '1 month',
+  },
+  {
+    label: m.securityDeposit.amountSelection2Month,
+    value: '2 months',
+  },
+  {
+    label: m.securityDeposit.amountSelection3Month,
+    value: '3 months',
+  },
+  {
+    label: m.securityDeposit.amountSelectionOther,
+    value: 'other',
+  },
+]
+
```

Comment:
Should this maybe be added to utils.ts?

Reply:
I don't like that pattern, since it's only used in this file and this contact.